lib/isorender/dom-shims.js

```diff
@@ -34,6 +34,7 @@ global.requestAnimationFrame = global.requestAnimationFrame || requestAnimationF
     const child = origFunc.apply(this, arguments);
     requestAnimationFrame(() => {
       if (!child.initialized && child.connectedCallback) {
+        child.isConnected = true;
```

Comment:
Now might be a good time to also patch `removeChild` etc to call `disconnectedCallback` in this environment...

Reply:
+1, if you feel like taking it on I'd love to see all of this functionality consolidated here in `dom-shims.js`: https://github.com/mixpanel/analytics/blob/master/iron/common/test/isotests/shims.ts#L363-L391

Reply:
doesn't need to block this PR fwiw, I'd be happy to collaborate separately if you prefer

Reply:
I'll take a stab at it 👍

Reply:
went ahead and plugged some of our attribute shims, attachShadow, and remove child here https://github.com/mixpanel/panel/pull/110/commits/f4da06345552d6e3ab823c23d3cba625c97225fc per @evnp's linked shims to analytics code. anything else we should add while at it?

Reply:
Thanks! that's great to have in here